build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+sh build-local.sh
+cp -r public-site/* ../docs/
```

Comment:
I don't think there's a need for the `public-site/` folder if we're copying it to the docs. We can probably remove this after it's been copied so we don't have any duplicates. With the comment above, we won't need the `build-local.sh` file. We can move the `tutors` command here to generate the site.